test/configCases/node/issue-18409-2/index.js

```
@@ -0,0 +1,8 @@
+import common from "./common";
+import separate from "./separate";
+
+it("should compile", () => {
+	expect(common).toBe("common");
+	expect(separate).toBe("separate");
+});
+export default "main";
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)